src/python/foglamp/device/coap/sensor_values.py

```diff
@@ -39,16 +41,30 @@ class SensorValues(aiocoap.resource.Resource):
     """CoAP handler for coap://readings URI"""
 
     _CONNECTION_STRING = "dbname='foglamp'"
+
     # 'postgresql://foglamp:foglamp@localhost:5432/foglamp'
 
+    _num_readings = 0
+    _num_discarded_readings = 0
```

Comment:
Please move these to init, initialize them as self._num_readings, and give them a short docstring.